docs/tutorial.rst

```diff
@@ -1,808 +0,0 @@
-========
-Tutorial
-========
```

Comment:
I'm guessing the tutorial may be a frequent target of inbound links (readthedocs has a rudimentary analytics page where we could possibly check). So we might want to figure out how to keep it as a redirect or link to something else.